src/Bridge/Doctrine/Orm/Filter/RangeFilter.php

```diff
@@ -102,7 +98,23 @@ private function addWhere(QueryBuilder $queryBuilder, QueryNameGeneratorInterfac
                 $rangeValue = explode('..', $value);
 
                 if (2 !== count($rangeValue)) {
-                    throw new InvalidArgumentException(sprintf('Invalid format for [%s], expected to be <min>..<max>', $operator));
+                    if (null !== $this->logger) {
+                        $this->logger->notice('Invalid filter ignored', [
+                            'exception' => new InvalidArgumentException(sprintf('Invalid format for "[%s]", expected "<min>..<max>"', $operator)),
+                        ]);
+                    }
+
+                    return;
+                }
+
+                if (!is_numeric($rangeValue[0]) || !is_numeric($rangeValue[1])) {
```

Comment:
Chaining conditions is not beneficial here because we still need to `return` early.